.travis.yml

```diff
@@ -20,7 +29,7 @@ branches:
 
 before_install:
   - (sudo apt-get update || brew update)
-  - (sudo apt-get install -y fontconfig libfribidi-dev yasm || (brew uninstall libtool; brew install fontconfig freetype fribidi libtool yasm)) && ./autogen.sh && ./configure
+  - (sudo apt-get install -y libfontconfig1-dev libfreetype6-dev libfribidi-dev libharfbuzz-dev yasm || brew install freetype fribidi harfbuzz yasm --without-gobject-introspection --without-icu4c) && ./autogen.sh && ./configure
```

Comment:
The Fontconfig and FreeType package names in Ubuntu contain ABI version numbers (?). We don’t really care about the ABI (only the API), so it seems that this might needlessly break in the future when a newer version is released but we keep requesting the old one. If we want, we can remove these two from this list, as both are (currently) preinstalled on Travis CI.